### ISSUE TITLE
Adding/removing nodes/edges after Sigma instance creation

### ISSUE THREAD

**Issue by gaardhus** [medialab/ipysigma#107]
Is it possible to add and remove nodes and edges after the instance has been created?

I tried:

```python
import networkx as nx
from ipysigma import Sigma

# Creating a dummy graph to serve as example
g = nx.karate_club_graph()

# Starting layout automatically
s = Sigma(g, start_layout=True)
s
```

But neither of

```python
s.data['nodes'].pop(0)
```
or
```python
s.graph.remove_node(0)
```
seemed to work.

**Comment by Yomguithereal** [comment]
Hello @gaardhus. I have not yet thought about this possibility indeed. Can you describe your use case a little bit more please? What are you trying to accomplish or visualize here? Is reloading the widget out of the question for the kind of thing you want to do?

**Comment by Yomguithereal** [comment]
@gaardhus any news on this?

**Comment by gaardhus** [comment]
Yeah sorry! Well the goal would be to visualize temporal/dynamic changes over time, by dropping/adding nodes/edges.

**Comment by SunnyPirate** [comment]
update?

**Comment by Yomguithereal** [comment]
Nope. This can be done, but this is far from trivial at least from a design perspective, since you need to choose if adding/dropping a node/edge at runtime should update the visual scales and such.
